gateway/api/use_cases/authentication.py

```diff
@@ -71,6 +72,10 @@ def execute(self) -> Optional[type[AbstractUser]]:
 
         access_groups = authentication_service.get_groups()
         quantum_user = self.user_repository.get_or_create_by_id(user_id=user_id)
+        if not UserAccessPolicies.can_access(quantum_user):
+            raise exceptions.AuthenticationFailed(
+                "Your user was deactivated. Please contact to support for reactivaton."
+            )
```

Comment:
Which support is this referring to? the function provider support? IBM support?

```suggestion
                "Your user was deactivated. Please contact support for reactivaton."
            )
```

Reply:
Good catch, IBM support I would say.